internal/restic/snapshot_test.go

```diff
@@ -14,3 +14,13 @@ func TestNewSnapshot(t *testing.T) {
 	_, err := restic.NewSnapshot(paths, nil, "foo", time.Now())
 	rtest.OK(t, err)
 }
+
+func TestTagList(t *testing.T) {
+	paths := []string{"/home/foobar"}
+	tags := []string{""}
+
+	sn, _ := restic.NewSnapshot(paths, nil, "foo", time.Now())
+
+	r := sn.HasTags(tags)
+	rtest.Assert(t, r, "Snapshot failed to match with non-empty selector")
```

Comment:
Sorry if I'm missing something here, but shouldn't this message be about failing to match with an *empty* selector, and/or perhaps even an empty *tag* instead of selector? The message seems a bit disconnected from what we're testing?